lib/diagnostic.rb

```diff
@@ -29,40 +29,40 @@
 
 character = 'Jar Jar Binks'
 
-Diagnostic.interpolation = 'ANSWER is really a Sith Lord.'
+Diagnostic.interpolation = puts '#{character} is really a Sith Lord.'
 
 # Define a predicate method in Ruby
 #
 # Define a method that checks whether a number passed in is odd. Name it
 # conventionally.
 
-# define your method here
+number_odd.odd?
```

Comment:
This works, but we were looking for a new method defined conventionally for a predicate, such as:

```
def num_odd? (n)
  # something with n
end
```